lldb-eval/eval_test.cc

```diff
@@ -2819,8 +3038,10 @@ TEST_F(EvalTest, TestCompositeAssignmentInvalid) {
   ASSERT_TRUE(CreateContextVariable("$i", "1"));
   EXPECT_THAT(EvalWithContext("($i += 1) -= 2", vars_),
               IsError("side effects are not supported in this context"));
+#endif
 }
 
+#ifndef __EMSCRIPTEN__
 TEST_F(EvalTest, TestCompositeAssignmentAdd) {
```

Comment:
Have you considered doing something like this when disabling the whole test?

```
#ifdef __EMSCRIPTEN__
  // Doesn't work on wasm32 because of X
  GTEST_SKIP() << "not supported on wasm32";
#else
```

Reply:
I did, but ifdef'ing out the entire test simplifies the wasm wrapper, so that it doesn't have to provide noop implementations of things like EvalWithContext() or Scope(). This seemed cleaner than implementing methods that aren't supposed to be called.